WinUIGallery/ControlPages/InfoBadgePage.xaml

```diff
@@ -0,0 +1,167 @@
+            </local:ControlExample.Xaml>
+        </local:ControlExample>
+
+        <local:ControlExample x:Name="Example2" HeaderText="Different InfoBadge Styles" HorizontalContentAlignment="Stretch" >
+            <local:ControlExample.Example>
+                <StackPanel Orientation="Horizontal" contract5Present:Spacing="20" HorizontalAlignment="Center">
+                    <InfoBadge x:Name="infoBadge2" Style="{StaticResource AttentionIconInfoBadgeStyle}" HorizontalAlignment="Right"/>
+                    <InfoBadge x:Name="infoBadge3" Style="{StaticResource AttentionValueInfoBadgeStyle}" HorizontalAlignment="Right" Value="10"/>
+                    <InfoBadge x:Name="infoBadge4" Style="{StaticResource AttentionDotInfoBadgeStyle}" VerticalAlignment="Center"/>
+                </StackPanel>
+
+            </local:ControlExample.Example>
+            <local:ControlExample.Options>
+                <StackPanel Width="160">
+                    <ComboBox x:Name="InfoBadgeStyleComboBox" Header="Styles" SelectionChanged="InfoBadgeStyleComboBox_SelectionChanged" SelectedValue="Attention">
+                        <x:String>Attention</x:String>
+                        <x:String>Informational</x:String>
+                        <x:String>Success</x:String>
+                        <x:String>Critical</x:String>
+                    </ComboBox>
+                </StackPanel>
+            </local:ControlExample.Options>
+
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;InfoBadge x:Name="infoBadge2" Style="{StaticResource $(Style)IconInfoBadgeStyle}" /&gt;
+&lt;InfoBadge x:Name="infoBadge3" Style="{StaticResource $(Style)ValueInfoBadgeStyle}" Value="10" /&gt;
+&lt;InfoBadge x:Name="infoBadge4" Style="{StaticResource $(Style)DotInfoBadgeStyle}" /&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="Style" Value="{x:Bind InfoBadgeStyleComboBox.SelectedValue, Mode=OneWay}" />
+            </local:ControlExample.Substitutions>
+        </local:ControlExample>
+
+        <local:ControlExample x:Name="Example3" HeaderText="Placing an InfoBadge Inside Another Control" HorizontalContentAlignment="Stretch" >
+            <local:ControlExample.Example>
+                <Button Padding="0" Width="200" Height="60" HorizontalAlignment="Center" HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch">
+                    <Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Width="Auto" Height="Auto">
+                        <SymbolIcon Symbol="Sync" HorizontalAlignment="Center"/>
+                        <InfoBadge Background="#C42B1C" HorizontalAlignment="Right" VerticalAlignment="Top">
+                            <InfoBadge.IconSource>
+                                <FontIconSource FontFamily="Segoe MDL2 Assets" Glyph="&#xF13C;"/>
```

Comment:
Font definitions like this should never be hardcoded. It's also old. Change it to StaticResource SymbolThemeIconFont

Reply:
@Jay-o-Way done